### PR TITLE
feat: replace font zoom in/out with scale  

### PR DESCRIPTION
**Pull request by apphiaWang**
In this PR:
- remove zoom in/out buttons from the toolbar
- add scale dropdown to control passage scale
- fix the drag rectangle
- add scale buttons (add/minus scale by 5%)
- update the scale range to [25%, 200%]
- override passage width on a large scale to avoid strophe buttons out of viewport

TBD:
- may need to update the override method

### CODE REVIEW

File: src/components/StudyPane/Passage/index.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const Passage = ({
     document.addEventListener('mouseup', handleMouseUp);
     setSelectionStart({ x: event.clientX + window.scrollX, y: event.clientY + window.scrollY });
     setSelectionEnd(null);
-
+
```
Comment (teamsela) [inline src/components/StudyPane/Passage/index.tsx:40]: Remove the unnecessary whitespace

File: src/components/StudyPane/Passage/WordBlock.tsx [hunk 2]
```diff
@@ -98,23 +100,23 @@ export const WordBlock = ({
   
   
     const verseNumStyles = {
-      className: `${zoomLevelMap[ctxZoomLevel].fontSize} top-0 ${ctxIsHebrew ? 'right-0' : 'left-0'} sups w-1 position-absolute ${ctxIsHebrew ? zoomLevelMap[ctxZoomLevel].verseNumMr : zoomLevelMap[ctxZoomLevel].verseNumMl}`
```
Comment (teamsela) [inline src/components/StudyPane/Passage/WordBlock.tsx:101]: Since we no longer use zoom level, it doesn't make sense to keep using the zoomLevelMap. We should just keep the font size and other sizes of level 5 and move from a map into an object.

Comment (teamsela) [reply src/components/StudyPane/Passage/WordBlock.tsx:101]: If you don't have time, you can just leave it as is. I can refactor the code later

**Review by teamsela — COMMENTED** [review]
The change looks great. Please add some padding between toolbar and passage and address my comments. Thanks!

File: src/components/StudyPane/Passage/WordBlock.tsx (hunk 2 above)
Comment (apphiaWang) [reply src/components/StudyPane/Passage/WordBlock.tsx:101]: I keep the zoom level map here because line 108 to 127 can use smaller font sizes to render words in the uniform width blocks.  Also line 106 uses `zoomLevel + 2` font size for Hebrew. Maybe we can discuss this Saturday then I can refactor this part.
